auto_nag/scripts/vacant_triage_owner.py

```diff
@@ -0,0 +1,167 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from jinja2 import Environment, FileSystemLoader
+from libmozdata.bugzilla import BugzillaProduct
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.nag_me import Nag
+from auto_nag.team_managers import TeamManagers
+from auto_nag.user_activity import UserActivity
+
+
+class TriageOwnerVacant(BzCleaner, Nag):
+    def __init__(self):
+        super(TriageOwnerVacant, self).__init__()
+        self.query_url = None
+
+    def description(self):
+        return "Components with triage owner need to be assigned"
+
+    def fetch_products(self):
+        data = []
+        include_fields = [
+            "name",
+            "is_active",
+            "components.id",
+            "components.name",
+            "components.team_name",
+            "components.triage_owner",
+            "components.is_active",
+        ]
+
+        def product_handler(product, data):
+            data.append(product)
+
+        BugzillaProduct(
+            product_names=self.get_products(),
+            include_fields=include_fields,
+            product_types=["accessible", "selectable", "enterable"],
+            product_handler=product_handler,
+            product_data=data,
+        ).wait()
+
+        return data
+
+    def nag_template(self):
+        return self.template()
+
+    def identify_vacant_components(self):
+        products = self.fetch_products()
+        triage_owners = set()
+        for product in products:
+            if not product["is_active"]:
```

Comment:
Is there a way to exclude these directly in the API call to get products?

Reply:
It seems no. 

In the docs, only the following are available:
- `id_or_name`
- `ids`
- `names`
- `type`

I tried using `is_active` in case it is available but not documented; unfortunately, it did not work.